lib/codecs/adts.js

```diff
@@ -119,20 +120,15 @@ AdtsStream = function(handlePartialSegments) {
         samplingfrequencyindex: (buffer[i + 2] & 0x3c) >>> 2,
         // assume ISO/IEC 14496-12 AudioSampleEntry default of 16
         samplesize: 16,
-        data: buffer.subarray(i + 7 + protectionSkipBytes, frameEnd)
+        data: buffer.subarray(i + 7 + protectionSkipBytes, i + frameLength)
```

Comment:
Would it make sense to keep the `frameEnd` variable and use it here and set `i` to equal `frameEnd` below (and above) just for clarity?

Reply:
From my point of view frameEnd just abstracts away what we are doing needlessly. I think that we should work in terms relative to i, rather than using an absolute value.

Reply:
👍

Reply:
I think the `frameEnd` variable can be removed in that case.